frontend/src/metabase-lib/lib/Dimension.js

```diff
@@ -542,6 +542,21 @@ export class FieldLiteralDimension extends FieldDimension {
       },
     });
   }
+
+  defaultDimension(): ?Dimension {
+    if (this.field().isDate()) {
+      return new DatetimeFieldDimension(
+        this,
+        // Field literals don't have the fingerprint data we use to pick this, so let's just arbitrarily go with "day".
```

Comment:
Which part of fingerprint would you need. In general even field literals should have fingerprint in `result_metadata`

Reply:
You're right the fingerprint data was there. Not sure how I missed it, but calling `getDefaultDateTimeUnit` does the right thing. Now the example query nicely groups by "month" rather than the arbitrarily chosen "day".